src/main/java/io/vertx/ext/sql/RowStreamWrapper.java

```diff
@@ -0,0 +1,86 @@
+package io.vertx.ext.sql;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonArray;
+
+import java.util.List;
+
+/**
+ * @author <a href="mailto:ruslan.sennov@gmail.com">Ruslan Sennov</a>
```

Comment:
Maybe a small doc with the use case

Reply:
I think this should be in the `impl` as it’s a wrapper that auto closes the connection not really an api to be used by the end user, right?

Reply:
Yes, this is the wrapper, it has package-visibility and is not intended for use by anyone. I see no difference if we move it to `impl` package or leave it here with  package-visibility. But if it is important, I can move it :)

Reply:
I understand but we do this as convention to explicitly tell users that internal APIs can change at any time and what’s in the main package is stable

Reply:
thank you, done